auth_oidc/readme/HISTORY.md

```diff
@@ -1,3 +1,7 @@
+## 16.0.1.3.0 2025-05-17
+
+- Add support for ES256, ES384, and HS256 algorithms in OpenID Connect
+
```

Comment:
This `HISTORY.md` is autogenerated on merge[^1]. Please add a files named `readme/newsfragment/797.feature` with a single paragraph explaining the change (no version nor date).

[^1]: we should add comment in it mentioning that!